senetence-gen

```diff
@@ -0,0 +1,8 @@
+name: sentence-gen
+desc: Generates silly sentences
+site: https://github.com/Awesome-Guy123/sentence-gen
+tags:
+  - python
+upforgrabs:
+  name: help make more templates
+  link: https://github.com/Awesome-Guy123/sentence-gen/blob/main/goals.md
```

Comment:
This link should point to a list of tasks under `https://github.com/Awesome-Guy123/sentence-gen/issues` - did you have any plans to create some issues and tag them with the right label?